package-lock.json

```diff
@@ -1,18348 +1,8 @@
 {
   "name": "radius-tracker",
   "version": "0.0.8",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
`v14.17.6`

Reply:
Weird. And what's your npm version?
Anyway, since there are no changes to `package.json`, seems that no changes to `package-lock.json` should be committed either.

Reply:
npm `6.14.15`. Are you sure, that you committed the last version of `package-lock.json`?

Reply:
I've got `8.1.2` — this explains why committed `lockfileVersion` is 2.
npm v6 uses `lockfileVersion` 1, npm v7+ uses `lockfileVersion` 2 or 3.

But I think my previous point stands — since there are no changes to `package.json`, an update to `package-lock.json` should not be committed either.